spec/spec_helper.rb

```diff
@@ -1,6 +1,7 @@
 require "bundler/setup"
 require "pg_ha_migrations"
 require "db-query-matchers"
+require "pry-byebug"
```

Comment:
This should be in the `.pryrc` file; if we don't have one we should add it.

Reply:
So we do have a `.pryrc` file with logic like this: https://github.com/braintree/pg_ha_migrations/blob/92ec97c9ffca4785b48ca84bb71afb914e827776/.pryrc#L1-L6

Googling around, I don't see a ton of examples that are requiring `pry-byebug` in this file